Doc/library/stdtypes.rst

```diff
@@ -2760,12 +2762,14 @@ arbitrary binary data.
             bytearray.decode(encoding="utf-8", errors="strict")
 
    Return a string decoded from the given bytes.  Default encoding is
-   ``'utf-8'``. *errors* may be given to set a different
-   error handling scheme.  The default for *errors* is ``'strict'``, meaning
-   that encoding errors raise a :exc:`UnicodeError`.  Other possible values are
-   ``'ignore'``, ``'replace'`` and any other name registered via
-   :func:`codecs.register_error`, see section :ref:`error-handlers`. For a
-   list of possible encodings, see section :ref:`standard-encodings`.
+   ``'utf-8'``.  And for a list of possible encodings, see section
+   :ref:`standard-encodings`.
```

Comment:
```suggestion
   *encoding* defaults to ``'utf-8'``;
   see :ref:`standard-encodings` for possible values.
```

As above:
* Per the original issue, give the *encoding* argument it's own paragraph for readability
* Explicitly state and format the parameter name at the beginning per the docs style guide
* Fix and greatly improve grammar/phrasing
* Be more concise and austere in reference docs